src/lib/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Picker } from "react-native"
+import styled from "styled-components/native"
+
+import { InvertedButton } from "../../Buttons"
+import Serif from "../../Text/Serif"
+
+export class SelectMaxBid extends React.Component<SelectMaxBidProps> {
+  render() {
+    return (
+      <Container>
+        <Serif style={{ fontSize: 20, textAlign: "center", fontWeight: "bold", margin: 18 }}>Your max bid</Serif>
```

Comment:
What is the convention around static components like this that don't really depend on props and have no children? Should we use styled components here?

Reply:
You probably will want the same formatting in some other screen too, so that makes sense

Reply:
Yeah I had the same question. @orta @mennenia do you have any wisdom on this?

Reply:
I think the ideal state is that the render method doesn't contain any styling, I know I've not succeeded in that in a bunch of places

On the other hand, I think it's fine to make a re-usable title component where it needs a custom bottom margin to just have that margin inlined, so you can understand where that change has come in from

Reply:
I would indeed make a `Title` styled component :). 

In Emission now we do try to steer away from inline style, unless, like Orta just said, it's a small margin indent. Though I would then still use a styled component that is based off something and adds that one property.

But you will very likely still see inline style throughout our codebase, from before we used Styled Comps / while we were still figuring things out.

Reply:
That header 👆 is the same on some other screens according to the [designs](https://app.zeplin.io/project/5ab94282c124db8d58365b47/screen/5acbdf943000a9041d8e9abe) so it could be reused.